### DISCUSSION TITLE
VPS features

### DISCUSSION THREAD

**Discussion by daysus** [engageub/InternetIncome#93] [Q&A]
What feature would the VPS have to have to run the monetizers allowed with docker? CPU, RAM, GBs of Space?

Any recommendation?

Thank you

**Comment by engageub** [comment]
It depends on what containers are being used and the number of instances being run.
You can start with 1 GB RAM and 1 CPU (frequency depends on the provider) without firefox containers and 10 GB of available disk space. Increase the swap space if necessary. 
You can then calculate and increase the corresponding numbers. 
The docker images are not owned by me, so it may change when there is update in the corresponding images. 

Thank you
